### PR TITLE
misc 2

### PR DESCRIPTION
**Pull request by jschueller**
- #2130  (rename to GALLIGAOGIBBS)
- #2139 
- #2140
- Deprecated legacy R plot
- #2144

### CODE REVIEW

**Review by sofianehaddad — APPROVED** [review]
Nice to drop the `R` legacy.

File: lib/src/Uncertainty/Process/openturns/GaussianProcess.hxx [hunk 1]
```diff
@@ -68,7 +68,7 @@ public:
   void setTimeGrid(const RegularGrid & timeGrid) override;
 
   /** Set sampling method accessor */
-  enum SamplingMethod { CHOLESKY, HMAT, GIBBS };
+  enum SamplingMethod { CHOLESKY, HMAT, GALLIGAOGIBBS };
```
[the remark this answers is not in the document]
Comment (jschueller) [reply lib/src/Uncertainty/Process/openturns/GaussianProcess.hxx]: ok

**Review by regislebrun — APPROVED** [review]
Bye bye R, an epoch is over...

**Comment by jschueller** [comment]
will miss you, or not :)

**Pull request merged**